packages/source-fetcher/lib/sourcify.ts

```diff
@@ -111,7 +111,8 @@ const SourcifyFetcher: FetcherConstructor = class SourcifyFetcher
     sourcePath: string
   ): Promise<string> {
     return await this.requestWithRetries<string>({
-      uri: `https://${this.domain}/contract/${this.networkId}/${address}/sources/${sourcePath}`
+      url: `https://${this.domain}/contract/${this.networkId}/${address}/sources/${sourcePath}`,
```

Comment:
In this case, the response is not JSON, so I believe you'll have to set `responseType: text`.

(IMO, it also might be a good idea to explicitly set `responseType: json` where you rely on that default, just for explicitness.)